src/query/sql/src/planner/optimizer/decorrelate/decorrelate.rs

```diff
@@ -486,3 +498,18 @@ impl SubqueryRewriter {
         }))
     }
 }
+
+// Check if contain group by in subquery
+fn contain_group_by(subquery: &SExpr) -> bool {
```

Comment:
it would be better to rename `subquery` to `s_expr`.

Reply:
It's only used by to check if contain group by in subquery